ModularTegustation/tegu_items/rcorp/itemspawner.dm

```diff
@@ -92,6 +92,26 @@
 		return
 	new spawning(get_turf(src))
 
+//Randomize the captain's rhino
+/obj/effect/landmark/rhinocaptainspawner
+	name = "rhino captain spawner"
+	desc = "It spawns an item. Notify a coder. Thanks!"
+	icon = 'icons/effects/landmarks_static.dmi'
+	icon_state = "x4"
+	var/list/possible_mecha = list(
+		/obj/vehicle/sealed/mecha/combat/rhino/captain,
+		/obj/vehicle/sealed/mecha/combat/rhinomelee/captain,
+		/obj/vehicle/sealed/mecha/combat/durand,
+	)
+
+/obj/effect/landmark/rhinocaptainspawner/Initialize()
+	..()
```

Comment:
Do not leave initialize without a return hint, always do eighter `. = ..()`, `return ..()` or sometimes `return INITIALIZE_HINT_[insert case here]`
```suggestion
	. = ..()
```